src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalToolHandler.java

```diff
@@ -97,9 +101,22 @@ private String getQueryParam(String key, String value) {
         if (hardCodedKludgeForDataExplorer) {
             if (key.equals("uri")) {
```

Comment:
Data Explorer requires `uri` as the key for a query parameter but we don't want this to be hard coded. I threw it in there when I started my "kludge" branch just to show the parts of the code that need to be modified.